forge-gui/res/cardsfolder/upcoming/excogitator_sphinx.txt

```diff
@@ -7,5 +7,5 @@ T:Mode$ DamageDoneOnce | ValidSource$ Creature.YouCtrl | TriggerZones$ Battlefie
 SVar:TrigInvestigate:DB$ Investigate
 A:AB$ Seek | Cost$ 1 Sac<1/Clue> | Type$ Instant,Sorcery | SpellDescription$ Seek an instant or sorcery card.
 DeckHas:Ability$Investigate|Token|Sacrifice & Type$Artifact|Clue
-DeckHints:Type$Clue
+DeckHints:Type$Clue|Sorcery|Instant
```

Comment:
Investigate makes a clue, right? So we should have some DeckHas too?

Maybe upgrade sorcery/instant to DeckNeeds? Is this card any good if that whiffs?